api/tests/routes/backoffice/pivots_test.py

```diff
@@ -655,3 +656,21 @@ def test_delete_pivot_ems(self, authenticated_client):
         self.post_to_endpoint(authenticated_client, name="ems", pivot_id=ems_pivot.id)
 
         assert not providers_models.EMSCinemaDetails.query.get(ems_pivot.id)
+
+    def test_delete_pivot_history_action(self, authenticated_client):
+        venue = offerers_factories.VenueFactory()
+        cgr_provider = providers_repository.get_provider_by_local_class("CGRStocks")
+        cinema_provider_pivot = providers_factories.CinemaProviderPivotFactory(
+            venue=venue,
+            provider=cgr_provider,
+            idAtProvider="idProvider1010",
+        )
+        cgr_pivot = providers_factories.CGRCinemaDetailsFactory(cinemaProviderPivot=cinema_provider_pivot)
+
+        response = self.post_to_endpoint(authenticated_client, name="cgr", pivot_id=cgr_pivot.id)
+        assert response.status_code == 303
+        assert not providers_models.CGRCinemaDetails.query.get(cgr_pivot.id)
+        action = history_models.ActionHistory.query.one()
+        assert action.comment == "Pivot CGR"
+        assert action.venueId == venue.id
+        assert action.actionType == history_models.ActionType.PIVOT_DELETED
```

Comment:
Tu peux vérifier aussi que l'auteur est bien dans l'historique : 
```suggestion
        assert action.actionType == history_models.ActionType.PIVOT_DELETED
        assert action.authorUser == legit_user
```
en ajoutant la fixture `legit_user` en paramètre de la méthode de test.